### PR TITLE
README: deleted bitcoin donation badge because the link does not work. P...

### PR DESCRIPTION
**Pull request by alexwoehr**
Per issue #946

Bitcoin donate link is broken in the README

### CODE REVIEW

**Comment by alexwoehr** [comment]
Added some updates to text on the main project as well.

Seems like my two pull requests got merged somehow. I am not sure how to disentangle them.

I will read over the main documentation later. My sense is that it has been read over pretty carefully, since it is the main project documentation that is viewed by users.

Hoping to do slow + steady work on this project. I am sick of excellent open source projects losing steam and want to do something about it.

File: locale/en.cson [hunk 1]
```diff
@@ -96,38 +96,38 @@
 	renderGeneratingNotification: "Website generating..."
 	renderGenerated: "Generated %s files in %s seconds"
 	renderGeneratedNotification: "Website generated"
-	renderParsing: 'Parsing everything'
```
Comment (balupton) [inline locale/en.cson]: ... are there for visual purposes, so you know that a task is still in progress

**Comment by balupton** [comment]
Other than the `...` thing, this PR is good to go.

**Comment by alexwoehr** [comment]
I backed out of this change in #947.

**Pull request closed without merging**